packages/nimble-components/build/generate-workers/source/matrix-renderer.ts

```diff
@@ -14,58 +16,69 @@ import type {
  * This setup is used in the wafer-map component to perform heavy computational duties
  */
 export class MatrixRenderer {
-    public columnIndexes = Int32Array.from([]);
-    public rowIndexes = Int32Array.from([]);
     public values = Float64Array.from([]);
     public scaledColumnIndex = Float64Array.from([]);
```

Comment:
We should rename `scaledColumnIndex` to `scaledColumnIndices` (and similarly `scaledRowIndex`). The term "indices" is used elsewhere in Nimble to refer to a collection of indexes (as opposed to just "indexes"). The only area where the term "indexes" is used is within production code is wafer-map related implementation.